### ISSUE TITLE
a question about training with/out VGenome

### ISSUE THREAD

**Issue by archanray** [Cadene/murel.bootstrap.pytorch#10]
Hi,
I trained using VGenome, but the performance on VQA overall went down. I used the supplied yaml as config, does anything needs to be changed?

**Comment by Cadene** [comment]
@LousyLory Thanks for your interest :)

- Which accuracy did you obtain with and without VGenome?
- On which benchmark (train/val, trainval/test-dev)?
- Is it the accuracy or the **open ended accuracy**?

**Comment by archanray** [comment]
Hi @Cadene, thanks for the response!

1. The new model under-performs in each of the categories as well as overall (yes/no, number, others and overall).
2. test-dev
3. MSCOCO-Open ended.

The following are the values I got:
                    without-VG      with-VG
yes/no              83.75            82.95
number             49.3             49.21
others              56.93             57.0
overall              67.11              66.8

I understand that these are within error rates, but the second model under-performs in each and every segment of the VQA challenge on test-dev data.

**Comment by Cadene** [comment]
@LousyLory Thanks for this info.

I am sorry for the issue.  We developed on pytorch v0.5 and released a clean code on pytorch v1.x. I will make sure that our results can be reproduce easily on this version as well. I will keep you updated.

While I focus in this issue, you could use the pretrained models of https://github.com/Cadene/block.bootstrap.pytorch as a baseline if you need to.

**Comment by archanray** [comment]
Cool thanks!!

**Comment by Zzmonica** [comment]
So which kind of accuracy did you get? Does it the MPT accuracy ? I got the results through the training procedure on VQA2 dataset. And the evaluation results are pretty low. I just upload the view.html to show that:
[view.zip](https://github.com/Cadene/murel.bootstrap.pytorch/files/3425812/view.zip)
